### PR TITLE
update structs to flint 2.9

### PR DESCRIPTION
**Pull request by tthsqe12**

### CODE REVIEW

File: src/antic/AnticTypes.jl [hunk 1]
```diff
@@ -50,10 +50,10 @@ function _AnticNumberField_clear_fn(a::AnticNumberField)
...
 
 mutable struct nf_elem <: SimpleNumFieldElem{fmpq}
-   elem_coeffs::Ptr{Nothing}
-   elem_den::Int
-   elem_alloc::Int
-   elem_length::Int
+   xelem_coeffs::Ptr{Nothing}
+   xelem_alloc::Int
+   xelem_length::Int
+   xelem_den::Int
```
Comment (thofma) [inline src/antic/AnticTypes.jl:56]: ?

Comment (tthsqe12) [reply src/antic/AnticTypes.jl:56]: oh may be not

**Comment by thofma** [comment]
I guess this includes #1292.

**Comment by tthsqe12** [comment]
Yup

**Pull request merged**